packages/console/src/pages/SignInExperience/components/ExperienceForm.tsx

```diff
@@ -0,0 +1,70 @@
+import { BrandingStyle, SignInExperience } from '@logto/schemas';
+import React from 'react';
+import { Control, Controller, UseFormRegister } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import FormField from '@/components/FormField';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import TextInput from '@/components/TextInput';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  register: UseFormRegister<SignInExperience>;
+  control: Control<SignInExperience>;
+};
+
+const ExperienceForm = ({ register, control }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <>
+      <div className={styles.title}>{t('sign_in_exp.branding.title')}</div>
+      <FormField isRequired title="admin_console.sign_in_exp.branding.primary_color">
+        {/* TODO: LOG-1733 color-picker */}
+        <TextInput {...register('branding.primaryColor', { required: true })} />
+      </FormField>
+      <FormField isRequired title="admin_console.sign_in_exp.branding.dark_mode">
+        {/* TODO: LOG-2152 switch */}
+        <TextInput {...register('branding.isDarkModeEnabled', { required: true })} />
+        <div>{t('sign_in_exp.branding.dark_mode_description')}</div>
+      </FormField>
+      <FormField isRequired title="admin_console.sign_in_exp.branding.dark_primary_color">
+        {/* TODO: LOG-1733 color-picker */}
+        <TextInput {...register('branding.darkPrimaryColor', { required: true })} />
+      </FormField>
+      <FormField isRequired title="admin_console.sign_in_exp.branding.ui_style">
+        {/* TODO: LOG-2153 plain radio */}
+        <Controller
+          name="branding.style"
+          control={control}
+          defaultValue={BrandingStyle.Logo_Slogan}
+          render={({ field: { onChange, value, name } }) => (
+            <RadioGroup value={value} name={name} onChange={onChange}>
+              <Radio value={BrandingStyle.Logo_Slogan}>
+                {t('sign_in_exp.branding.app_logo')} + {t('sign_in_exp.branding.text')}
```

Comment:
Should be `Logo` & `Slogan`

<img width="370" alt="image" src="https://user-images.githubusercontent.com/10594507/163102213-df41245c-0464-4a00-a7b1-50a18cf12888.png">

Reply:
updated